welcome-to-sourcery.py

```diff
@@ -1,49 +1,33 @@
+# Welcome to Sourcery! We're here to be your pair programmer anytime you're
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Should we keep the blank first line?

I know in Intellij it can look better with a blank first line because it avoids the end of the line being hidden by problems:
<img width="511" alt="image" src="https://github.com/sourcery-ai/sourcery-vscode/assets/1440886/3ed01ed4-0347-4afb-ba30-d6f2705af532">

Maybe it could be split into a title and headline?
```python
# Welcome to Sourcery!

# We're here to be ...
```

Reply:
We originally had it in VS Code because we had the token input box appearing at the top, which would obscure it. So I think we don't need it in VS Code at least.